### DISCUSSION TITLE
Some elements of web ui randomly becoming unresponsive

### DISCUSSION THREAD

**Discussion by Ilija712** [TheLastBen/fast-stable-diffusion#1898] [Q&A] [answered]
I haven't used stable diffusion in a while for about 2-3 weeks. Decided yesterday to generate some images but from time to time elements of stable diffusion web ui just become unresponsive. Sometimes its the generate button or save button or high res checkbox. Have no idea if this is me problem or web ui problem or notebook problem or what. Any help appreciated. Also sometimes images generate but not show in the ui but I can find them in the output directory in colab. Thanks!

**Comment by TheLastBen** [comment, marked as answer]
use ngrok to avoid freezing issues

**Reply by TheLastBen** [reply]
get a token here https://dashboard.ngrok.com/get-started/your-authtoken and paste it in the last cell

**Reply by ClaraSanders** [reply]
This happens to me from time to time. Now I solved it by turning off cloudflare tunnel without using ngrok.
